scipy/sparse/csgraph/_shortest_path.pyx

```diff
@@ -168,6 +168,60 @@ def shortest_path(csgraph, method='auto',
     >>> predecessors
     array([-9999,     0,     0,     1], dtype=int32)
 
+    >>> graph = [
+    ... [0, 0, 7, 0],
+    ... [0, 0, 8, 5],
+    ... [7, 8, 0, 0],
+    ... [0, 5, 0, 0]
+    ... ]
+    >>> graph = csr_array(graph)
+    >>> print(graph)
```

Comment:
Let's remove this print, the previous example shows what csr array looks like when printed